integration-tests/src/test/resources/embedded-schema.sql

```diff
@@ -0,0 +1,57 @@
+DROP Table IF Exists message;
```

Comment:
There should be no need to duplicate this schema.
The scripts that you need should be on the classpath, e.g.  https://github.com/eventuate-tram/eventuate-tram-core/blob/master/eventuate-tram-in-memory/src/main/resources/eventuate-tram-embedded-schema.sql and https://github.com/eventuate-tram/eventuate-tram-sagas/blob/master/eventuate-tram-sagas-in-memory/src/main/resources/eventuate-tram-sagas-embedded.sql

Reply:
ok